packages/bot-web-ui/src/components/contract-card-loading/contract-card-loading.tsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import ContentLoader from 'react-content-loader';
 
-const ContractCardLoader: React.FC = () => (
-    <ContentLoader height={153} width={334} speed={3}>
+type TContractCardLoader = {
+    speed: number;
+};
+
+const ContractCardLoader: React.FC<TContractCardLoader> = ({ speed }) => (
+    <ContentLoader height={153} width={334} speed={speed ?? 3}>
```

Comment:
```suggestion
const ContractCardLoader = ({ speed = 3 }: TContractCardLoader ) => (
    <ContentLoader height={153} width={334} speed={speed}>
```
Since there is a default value, it is cleaner to declare it within the props. You can make the type optional as it has a default value by behaviour. Not sure how the bot project does things but `explicit` declaration using `React.FC` is currently not recommended. https://spin.atomicobject.com/2022/01/04/think-twice-react-fc/

Reply:
fixed